packages/excalidraw/components/App.tsx

```diff
@@ -5097,8 +5097,8 @@ class App extends React.Component<AppProps, AppState> {
 
         this.translateCanvas({
           zoom: zoomState.zoom,
-          scrollX: zoomState.scrollX + deltaX / nextZoom,
-          scrollY: zoomState.scrollY + deltaY / nextZoom,
+          scrollX: zoomState.scrollX + 2 * (deltaX / nextZoom),
+          scrollY: zoomState.scrollY + 2 * (deltaY / nextZoom),
```

Comment:
why 2x? Isn't this more related to `devicePixelRatio` which on most mobile touchscreens would coincide with `2`?